types/dragino/lse01/uplink.js

```diff
@@ -0,0 +1,40 @@
+function hexToBytes(hex) {
+  for (var bytes = [], c = 0; c < hex.length; c += 2) {
+    bytes.push(parseInt(hex.substr(c, 2), 16));
+  }
+  return bytes;
+}
+
+function consume(event) {
+  const payload = event.data.payload_hex;
+  const bytes = hexToBytes(payload);
+  const data = {};
+  const lifecycle = {};
+
+  let value = ((bytes[0] << 8) | bytes[1]) & 0x3fff;
```

Comment:
prevent reusing this variable for different calculations, rather use separate variables